.github/workflows/_restart_api.yml

```diff
@@ -0,0 +1,67 @@
+name: Reusable Deploy based on scripts
```

Comment:
OOC - why is the naming convention here to prefix with "Reusable Deploy"?

Would simply "Restart API" be better?

Reply:
done